Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM phusion/baseimage:0.9.18
+
+MAINTAINER Luke Campbell <luke.campbell@rpsgroup.com>
+
+RUN apt-get update && apt-get install -y \
+      git \
+      libffi-dev \
+      libhdf5-dev \
+      libnetcdf-dev \
+      netcdf-bin \
+      libssl-dev \
+      libxml2-dev \
+      libxslt1-dev \
+      nodejs \
+      npm \
+      python-dev \
+      python-pip \
+      libudunits2-0 \
+      redis-tools \
+      libgeos-dev \
+      libpng12-dev \
+      libfreetype6-dev \
+      && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
+# The debian version of six caused a bug in Flask
+RUN pip install -U 'numpy==1.11.0' 'netCDF4==1.2.4'
+RUN pip install -U six
+RUN pip install cc-plugin-ncei
```

Comment:
I felt like the plugin wasn't a requirement for the project but it is a REALLY nice to have in the docker container.